index.js

```diff
@@ -15,14 +15,22 @@ module.exports = {
     },
     sourceType: "module"
   },
-  plugins: ["prettier"],
+  plugins: ["prettier", "import"],
   globals: {
     __DEV__: true
   },
   rules: {
     // Prettier
     "prettier/prettier": ["error", { printWidth: 110, singleQuote: true }],
 
+    // Import
+    "import/order": ["error", { groups: [["builtin", "external", "internal"]] }],
```

Comment:
in react we have a different rule, but I think it's fine to have this one as the general one